src/Sulu/Bundle/ContentBundle/Content/Structure/ExcerptStructureExtension.php

```diff
@@ -90,7 +90,10 @@ public function __construct(
      */
     public function save(NodeInterface $node, $data, $webspaceKey, $languageCode)
     {
-        foreach ($this->excerptStructure->getProperties() as $property) {
+        $excerptStructure = $this->getExcerptStructure();
```

Comment:
hmm a structure is always locale-less (: so we have to adapt the locale every time! we could remove the caching in this class because of caching in the structuremanager. therefor both would be ok for me? should i remove the caching in here completely getting new instances every time or adapt the cached value (like i have done it in this PR)? getting new instances every time seems to be more "saver" because we pass it to other services and they could adapt the structure and corrupt it ...